### PR TITLE
E-936 feat: choose doc

### PR DESCRIPTION
**Pull request by dudiq**

### CODE REVIEW

**Comment by dudiq** [comment]
@lamartire ready for review

File: src/plugins/OauthPlugin/Oauth/Oauth.js [hunk 1]
```diff
@@ -7,7 +7,9 @@ import Polling from '@/plugins/OauthPlugin/Oauth/Polling';
 
 const { ERRORS } = ConnectError;
 
-const AUTH_STATUS_KEY = 'endpass-oauth-hash';
+const DEFAULT_KEY = 'endpass-oauth';
```
[the remark this answers is not in the document]
Comment (dudiq) [reply src/plugins/OauthPlugin/Oauth/Oauth.js:10]: this is local constants and no need to define them in all constants

File: src/plugins/OauthPlugin/Oauth/Oauth.js [hunk 2]
```diff
@@ -25,8 +27,8 @@ export default class Oauth {
   /**
    * @returns {string}
```
[the remark this answers is not in the document]
Comment (dudiq) [reply src/plugins/OauthPlugin/Oauth/Oauth.js:28]: what do you mean? 
current line already have `@returns`

File: src/class/DialogView/Styles.js [hunk 3]
```diff
@@ -51,3 +57,8 @@ export const propsIframeHide = {
 export const propsIframeShow = {
   opacity: 1,
 };
+
+export const propsIframeFullScreen = {
```
[the remark this answers is not in the document]
Comment (dudiq) [reply src/class/DialogView/Styles.js:61]: for iframe we doesn't use `inlineStyles`, we are using `frameStyles`

File: src/plugins/OauthPlugin/Oauth/Oauth.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (dudiq) [reply src/plugins/OauthPlugin/Oauth/Oauth.js]: my fault, did not read correctly message, fixed

**Comment by kazurus** [comment]
approved

**Pull request merged**